File: longestWord/longestWord.py
def longestWord(string):
    length =0
    #lonWord = ""
    words = []
    stringList = string.split(" ")
    for word in stringList: #iterate over stringList array
        if len(word)> length: #check lengths
            words = []
            words.append(word)
            #lonWord = word
            length = len(word)
        elif len(word)== length:
            words.append(word)
    return ",".join(words) #join words with comma if words have same maximum length
print(longestWord(str(input("Enter a string: ")))) #print longest word or words in string

            
        

File: longestWord/longestWord.java
public class longestWord 
{
    public static void main(String[] args) 
    {
        String s="find the longest word";
        String[] word=s.split(" ");
        String longword=" ";
        for(int i=0;i<word.length;i++){
            for(int j=1+i;j<word.length;j++){
                if(word[i].length()>=word[j].length()){
                    longword=word[i];
                }
            }
        }
        System.out.println(longword+" is longest word with "+longword.length()+" characters");
        System.out.println(rts.length());
    }
}